application-templates/typescript/service/src/connector/actions.ts

```diff
@@ -0,0 +1,130 @@
+import { ByProjectKeyRequestBuilder } from '@commercetools/platform-sdk/dist/declarations/src/generated/client/by-project-key-request-builder';
+
+const CART_UPDATE_EXTENSION_KEY = 'myconnector-cartUpdateExtension';
+const CART_DISCOUNT_TYPE_KEY = 'myconnector-cartDiscountType';
+
+export async function createCartUpdateExtension(
+  apiRoot: ByProjectKeyRequestBuilder,
+  applicationUrl: string
+): Promise<void> {
+  const list = await apiRoot
```

Comment:
Give this a better-explaining name?

```suggestion
  const extensionsList = await apiRoot
```
or,

```suggestion
  const existingExtensionsList = await apiRoot
```

Reply:
Thanks, @ruidias-commercetools, it is much better now 😃 I used the object-destructing approach to get the extension model instead but the idea is the same.